sdk/helper/pluginutil/run_config.go

```diff
@@ -143,9 +159,15 @@ func (rc runConfig) containerConfig(env []string) *plugincontainer.Config {
 		GroupAdd: os.Getgid(),
 		Runtime:  consts.DefaultContainerPluginOCIRuntime,
 		Labels: map[string]string{
-			"managed-by": "hashicorp.com/vault",
+			"managed-by":            "hashicorp.com/vault",
```

Comment:
I reckon delete this one? It's covered (albeit implicitly) by the other label names, and they have much better structure.

```suggestion
```

Reply:
Sure, removed in 6a8700b759